tests/test_iterable_dataset.py

```diff
@@ -841,6 +877,27 @@ def test_iterable_dataset_map_with_features(dataset: IterableDataset) -> None:
     assert dataset.info.features == features_after_map
 
 
+def test_iterable_dataset_map_with_fn_kwargs(dataset: IterableDataset) -> None:
+    fn_kwargs = {"y": 1}
+    with dataset.map(lambda x, y: {"id+y": x["id"] + y}, fn_kwargs=fn_kwargs) as mapped_dataset:
```

Comment:
No need to use `with` statements on `IterableDataset` since it doesn't use temporary files when `map` is called. You can remove the `with` statements in this test and the next one.

Reply:
Thanks! I have modified them. 9bed249f8f33daff13c24b3d8a3c7cc95124eb1f